.gitignore

```diff
@@ -45,3 +45,5 @@ dependency-reduced-pom.xml
 *.iws
 docs-source/docs
 .git/logs
+docs/charts/index.yaml
```

Comment:
Why did you add this?  We set up the build to keep the helm chart repository files up-to-date.

Reply:
I had noticed that every time you do a build for testing, it wants to push your changes to this file to git, even if you make no changes to the chart. A fair number of PRs have included what seem to be extraneous changes like this. 

I think we have a philosophy difference, here. The standard way of using Maven is to distinguish between source and build. You check in source code, and do a build for test and release. We're combining them, and maybe I don't understand your approach, as it seems error-prone. Can you explain?

Reply:
These files are built from the chart, so the file is rebuilt anytime that files in the chart change.  I'm happy to hear ideas to improve the build process, but not as part of an unrelated PR.